framework/src/Volo.Abp.AspNetCore.Mvc.UI.MultiTenancy/Volo/Abp/AspNetCore/Mvc/UI/MultiTenancy/Localization/en.json

```diff
@@ -2,7 +2,7 @@
   "culture": "en",
   "texts": {
     "GivenTenantIsNotExist": "Given tenant does not exist: {0}",
-    "GivenTenantIsNotAvailable": "Given tenant is not available: {0}",
+    "GivenTenantIsNotAvailable": "Given tenant does not available: {0}",
```

Comment:
`is` instead of `does` is the correct usage since `available` isn't a verb here.